src/comment/comment.controller.ts

```diff
@@ -27,13 +27,41 @@ export class CommentController {
   
 
   @Get()
-  findAll() {
-    return this.commentService.findAll();
+  @ApiOperation({ summary: 'Get all comments' })
+  @ApiQuery({ name: 'postId', required: false })
+  async findAll(@Query('postId') postId: string, @Res() res) {
+    try {
+      const comments = await this.commentService.findAll(postId);
+      return res.status(HttpStatus.OK).json({
+        message: 'Comments has been successfully retrieved',
+        comments
+      });
+    } 
+    catch (error) {
+      return res.status(HttpStatus.BAD_REQUEST).json({
+        statusCode: HttpStatus.BAD_REQUEST,
```

Comment:
tại sao ở trên đã res.status(httpstatus.bad_requeset r ở dưới này vẫn phải trả về status code

Reply:
It's simple, res.status(HttpStatus.BAD_REQUEST) is return to HTTP response, and statusCode: HttpStatus.BAD_REQUEST is to return restAPI

Reply:
I have fix according to feedback, Please merge this branch